src/main/java/com/leinardi/pycharm/mypy/mpapi/MypyRunner.java

```diff
@@ -335,7 +335,8 @@ public static String detectSystemMypyPath() {
                     String[] splitPosition = rawLine.substring(0, typeIndexStart - 1).split(":", -1);
                     String path = splitPosition[0].trim();
                     int line = splitPosition.length > 1 ? Integer.parseInt(splitPosition[1].trim()) : 1;
-                    int column = splitPosition.length > 2 ? Integer.parseInt(splitPosition[2].trim()) : 1;
+                    // Mypy uses 1-based column numbers, IntelliJ expects 0-based
+                    int column = splitPosition.length > 2 ? Integer.parseInt(splitPosition[2].trim()) - 1 : 1;
```

Comment:
Alternatively this subtraction could be done in `com.leinardi.pycharm.mypy.mpapi.ProcessResultsThread#findPosition`. Not sure which is preferable.

Reply:
I think here is fine :+1: